src/components/material/digital-modal/DigitalModalError.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
The only difference between the DigitalModalSuccess and the DigitalModalError is the description. I think you could just make one component that could be called DigitalModalFeedback or similar?